dashboard/test/ui/features/step_definitions/eyes_steps.rb

```diff
@@ -8,6 +8,10 @@
 # See http://support.applitools.com/customer/en/portal/articles/2099488-match-timeout
 MATCH_TIMEOUT = 5
 
+# Prefix to print before any eyes errors. We depend on this in runner.rb to detect
+# whether there were any eyes errors.
+EYES_ERROR_PREFIX = '[Applitools Eyes error]'
```

Comment:
Could we load this constant from a shared location in both eyes_steps.rb and runner.rb?

Reply:
but then I'd have to pick a name for the file this gets extracted into :-) PTAL